### PR TITLE
Fix: Scrollbar on autosized textarea

### PR DESCRIPTION
**Pull request by iamstiil**
Fix for #13161
Tested on: Chrome 71

### CODE REVIEW

File: src/lib/form-field/form-field-input.scss [hunk 1]
```diff
@@ -127,13 +127,6 @@ textarea.mat-input-element {
   }
 }
 
-textarea.mat-input-element {
-  // The 2px padding prevents scrollbars from appearing on Chrome even when they aren't needed.
```
Comment (crisbeto) [inline src/lib/form-field/form-field-input.scss]: It seems like these changes were added to fix a different issue. @mmalerba do you know whether they can removed now? You show up on the blame for the changes.

Comment (crisbeto) [reply src/lib/form-field/form-field-input.scss]: Removing the styles ends up with a wrong height on my end as well @mmalerba. It also shows scrollbars when it isn't supposed to.

Comment (pavethiran) [reply src/lib/form-field/form-field-input.scss]: @mmalerba I am using Windows 10 Pro at standard zoom level.

**Comment by mmalerba** [comment]
Closing due to inactivity

**Pull request closed without merging**